DependencyInjection/DoctrineExtension.php

```diff
@@ -927,4 +936,25 @@ private function createArrayAdapterCachePool(ContainerBuilder $container, string
 
         return $id;
     }
+
+    private function registerMetadataPhpArrayCaching(string $entityManagerName, ContainerBuilder $container): void
+    {
+        $metadataCacheAlias              = $this->getObjectManagerElementName($entityManagerName . '_metadata_cache');
+        $decoratedMetadataCacheServiceId = (string) $container->getAlias($metadataCacheAlias);
+        $phpArrayCacheDecoratorServiceId = $decoratedMetadataCacheServiceId . '.php_array';
+        $phpArrayFile                    = '%kernel.cache_dir%' . sprintf('/doctrine/orm/%s_metadata.php', $entityManagerName);
+
+        $container->register(DoctrineMetadataCacheWarmer::class)
```

Comment:
@ossinkine actually this is not working properly for multiple entity managers. I just tried it on one of my bigger projects that uses 2 managers.

Since `DoctrineMetadataCacheWarmer::class` is the service id it will only work for the last manager and overwrite any previous definitions